internal/app/service.go

```diff
@@ -67,6 +70,10 @@ const (
 	metricTargetType = "metric"
 	eventTargetType  = "event"
 	emptyTargetType  = ""
+
+	messageBusDisabledErr = "publish failed due to MessageBus disabled via configuration"
+	publishDataErr        = "failed to publish data to messagebus"
```

Comment:
Please scan code for same change needed in other comments and messages
```suggestion
	publishDataErr        = "failed to publish data to MessageBus "
```